sktime/registry/_scitype.py

```diff
@@ -7,6 +7,15 @@
 from sktime.registry._base_classes import get_base_class_register
 
 
+def is_a_scitype(obj, desired_scitype):
```

Comment:
please add a docstring (or ask copilot to add one 😁 )

Also, I would allow `desired_scitype` to be a tuple, and return true iff there is non-trivial intersecion between the tuple and the list

Reply:
done